README.md

```diff
@@ -131,9 +131,15 @@ twyn run --config <file>
 All the configurations available through the command line are also supported in the config file.
 
 ```toml
-  [tool.twyn]
-  dependency_file="/my/path/requirements.txt"
-  selector_method="first_letter"
-  logging_level="debug"
-  allowlist=["my_package"]
+[tool.twyn]
+dependency_file="/my/path/requirements.txt"
+selector_method="first_letter"
+logging_level="debug"
+allowlist=["my_package"]
+pypi_reference="https://mirror-with-trusted-dependencies.com/file.json"
 ```
+
+> [!WARNING]
```

Comment:
should we indicate somewhere the format of the json that we're expecting?

Reply:
Indeed, I will just link to the default URLs for each reference then, and indicate that you should have the same file format.

Reply:
Changed, and also changed my mind. I added the interface we need from the json.